modin/pandas/test/test_benchmarks.py

```diff
@@ -0,0 +1,213 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
What about `ci/benchmarks` as a place for all performance test?

Reply:
> What about `ci/benchmarks` as a place for all performance test?

I don't think they are used anywhere.

Reply:
It might be nice to separate benchmarks from unit tests.

Reply:
What is your suggestion? Move this file into `benchmarks` subfolder?

Reply:
Yes, I believe that is also @anmyachev's suggestion as well.

Reply:
Ok I don't insist on this location and moved this file to `ci/benchmarks`.
Other files in that place however use their own test framework and aren't pytest based.